pints/_log_pdfs.py

```diff
@@ -115,6 +115,178 @@ def sample(self, n=1):
         raise NotImplementedError
 
 
+class PooledLogPDF(LogPDF):
+    r"""
+    Creates an object of :class:LogPDF, with a log-pdf which is the sum of the
+    individual log-pdfs, while pooling selected parameters.
+
+    This :class:`LogPDF` requires that all :class:`LogPDF` objects have the
+    same number of parameters.
+
+    This is useful for e.g. modelling the time-series of multiple individuals
+    (each individual defines a separate :class:`LogPDF`), and some parameters
+    are expected to be the same across individuals (for example, the noise
+    parameter across different individuals within the same experiment).
+
+    For two :class:`LogPDF` objects :math:`L _1`  and :math:`L _2` with
+    parameters :math:`(\psi _1, \theta _1`) and :math:`(\psi _2, \theta _2`)
+    respectively, a pooling of the :math:`\theta _i` results in a pooled
+    log-pdfs of the form
+
+    .. math::
+        \log L(\psi _1, \psi _2, \theta | D_1, D_2) =
+            \log L(\psi _1, \theta | D_1) + \log L(\psi _2, \theta | D_2),
+
+    where :math:`\theta := \theta _1 = \theta _2`, and $D_i$ is the measured
+    time-series of individual :math:`i`.
+
+    In general, the search of a :class:`PooledLogPDF` has dimensionality
+    :math:`nd_{\text{up}} + d_{\text{p}}`, where :math:`n` is the number of
+    individuals, :math:`d_{\text{up}}` is the dimension of the unpooled
+    :math:`psi` parameters, and :math:`d_{\text{up}}` is the dimension of the
+    pooled :math:`\theta` parameter.
+
+    The order of parameters for evaluation is largely kept, and the parameters
```

Comment:
"is largely kept" sounds to vague for me! Just write a precise description (like below) :-)